Casks/g/gitfox.rb

```diff
@@ -9,12 +9,14 @@
   homepage "https://www.gitfox.app/"
 
   livecheck do
-    url "https://storage.googleapis.com/gitfox/Gitfox.latest.stable.zip"
-    strategy :extract_plist
+    url "https://api.gitfox.app/v1/versions"
+    strategy :json do |json|
+      json["response"]["builds"].map { |item| "#{item["short_version_string"]},#{item["build"]}" }
```

Comment:
```suggestion
      json.dig("response", "builds")&.map do |item|
        next unless item["short_version_string"] && item["build"]

        "#{item["short_version_string"]},#{item["build"]}"
      end
```

Using `#dig` for multi-level values allows us to use the safe navigation operator after it. This suggestion also accounts for potential `nil` values in the `#map` block, so we don't end up with partial versions (e.g., `,1234`, `1.2.3,`, `,`) if expected values are missing.

Reply:
Thanks!